### ISSUE TITLE
Setting a default

### ISSUE THREAD

**Issue by chearmstrong** [opentable/accept-language-parser#25]
Hi,

Is it possible to have a default language fallback - for example, if the accept-language header doesn't contain one of our supported languages it uses a default. For example:

```parser.pick(['en-GB'], 'en-US'); // can it return en-GB not null```

Thanks.

**Comment by ArnoldZokas** [comment]
Do you mean something like this?

```javascript
var supportedLanguages = ['en-GB'];
var parsedAcceptLanguageHeader = 'en-US';
var defaultLanguage = 'en-GB';

parser.pick(supportedLanguages, parsedAcceptLanguageHeader) || defaultLanguage;
```

**Comment by chearmstrong** [comment]
Yeah, that's what I'm doing at the moment. I was just wondering if `parser.pick` could do it. However, it's no biggie. I was thinking more if someone forgot to set the default.
